core/src/main/java/com/linecorp/armeria/internal/server/websocket/WebSocketServiceUtil.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.server.websocket;
+
+public final class WebSocketServiceUtil {
```

Comment:
If this will also be used from client-side, it might be worth moving this class under `internal/common/websocket`. I'm fine with the current location though

Reply:
Fixed. 😉